docs/data_dictionaries/index.rst

```diff
@@ -3,20 +3,38 @@
 Data Dictionaries
 =================
 
+Data Processed & Cleaned by PUDL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The PUDL data dictionary provides detailed metadata for core and output tables in the PUDL database,
+including field names, data types, and descriptions
+for each field by table.
```

Comment:
This wording might accidentally imply that there are other tables in the PUDL database that are not core or output tables that do not have metadata here. To avoid confusion, would say: 

"The PUDL data dictionary provides detailed metadata for the tables in the PUDL database. This includes table descriptions, field names, field, descriptions, and field datatypes."